@uportal/content-carousel/src/components/ContentCarousel.ts

```diff
@@ -20,8 +20,8 @@ export default class ContentCarousel extends Vue {
   @Prop({ type: [String, Object], default: () => ({}) })
   public slickOptions!: string | Object;
 
-  @Prop({type: [String], default: 'auto'})
-  public carouselHeight?: string;
+  @Prop({ type: [String], default: 'auto' })
+  public carouselHeight!: string;
```

Comment:
:notebook: because a default is provided, `carouselHeight` should never be `undefined` or `null`